src/pages/TokenDetails/TokenAccounts.tsx

```diff
@@ -44,152 +30,47 @@ export const TokenDetailsAccounts = () => {
 
   const [accounts, setAccounts] = useState<AccountType[]>([]);
   const [accountsCount, setAccountsCount] = useState(0);
-  const [dataReady, setDataReady] = useState<boolean | undefined>();
+  const [isDataReady, setIsDataReady] = useState<boolean | undefined>();
 
   const fetchAccounts = () => {
     Promise.all([
       getTokenAccounts({ tokenId: identifier, page, size }),
       getTokenAccountsCount({ tokenId: identifier })
     ]).then(([tokenAccountsData, tokenAccountsCountData]) => {
-      if (ref.current !== null) {
-        if (tokenAccountsData.success && tokenAccountsCountData.success) {
-          setAccounts(tokenAccountsData.data);
-          setAccountsCount(tokenAccountsCountData.data);
-          setDataReady(true);
-        }
+      if (tokenAccountsData.success && tokenAccountsCountData.success) {
+        setAccounts(tokenAccountsData.data);
+        setAccountsCount(tokenAccountsCountData.data);
       }
+      setIsDataReady(
```

Comment:
the same question here (related to error handling)

Reply:
https://github.com/multiversx/mx-explorer-dapp/pull/132#discussion_r1806106196